packages/dataclass-editor/src/utils/browser/typeBrowserUtils.test.ts

```diff
@@ -0,0 +1,112 @@
+import { describe, expect } from 'vitest';
+import { getInitialSelecteState, getInitialTypeAsListState, getInitialValue } from './typeBrowserUtils';
+import type { BrowserNode } from '@axonivy/ui-components';
+import type { DataclassType } from '../../protocol/types';
+import { IvyIcons } from '@axonivy/ui-icons';
+
+describe('Utility Functions', () => {
```

Comment:
not needed. only makes the test name longer and adds one more step on the test tree...